test/tests/smoke_test.rs

```diff
@@ -50,4 +99,53 @@ fn smoke_test() {
 | |__| (_| | | | |_) | |_| | | (_| | |  _ < | |
  \____\__,_|_|_| .__/ \__|_|  \__,_| |_| \_\|_|"#,
     );
+
+    const TEST_ONLY_GET_LDEV_CERT: u32 = 0x4345524c; // "CERL"
+    const TEST_ONLY_GET_FMC_ALIAS_CERT: u32 = 0x43455246; // "CERF"
+
+    let ldev_cert_der = hw
+        .mailbox_execute(TEST_ONLY_GET_LDEV_CERT, &[])
+        .unwrap()
+        .unwrap();
+    let ldev_cert = openssl::x509::X509::from_der(&ldev_cert_der).unwrap();
+    let ldev_cert_txt = String::from_utf8(ldev_cert.to_text().unwrap()).unwrap();
+
+    // To update the ldev cert testdata:
+    // std::fs::write("tests/smoke_testdata/ldevid_cert.txt", &ldev_cert_txt).unwrap();
+    // std::fs::write("tests/smoke_testdata/ldevid_cert.der", &ldev_cert_der).unwrap();
```

Comment:
is this test basically validating the DUT against  n-1 DUT ?

Reply:
Yes, where n is a git commit. These certs and CSRs are generally not expected to change, so we want to make sure the test makes some noise if they do unexpectedly.